### PR TITLE
LG-3721 Disable Chromatic for Popover components

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

Chromatic tests for these components are flaky. Temporarily disabling them

🎟 _Jira ticket:_ https://jira.mongodb.org/browse/LG-3721

### CODE REVIEW

File: packages/guide-cue/src/GuideCue.story.tsx [hunk 1]
```diff
@@ -40,6 +40,10 @@ const meta: StoryMetaType<any> = {
         'onPrimaryButtonClick',
       ],
     },
+    chromatic: {
+      disableSnapshot: true,
+      delay: transitionDuration.slowest,
```
Comment (bruugey) [inline packages/guide-cue/src/GuideCue.story.tsx:45]: Do we need this on Menu too?

Comment (bruugey) [reply packages/guide-cue/src/GuideCue.story.tsx:45]: And SplitButton?

Comment (TheSonOfThomp) [reply packages/guide-cue/src/GuideCue.story.tsx:45]: Don't need the delay if we're disabling it

Comment (bruugey) [reply packages/guide-cue/src/GuideCue.story.tsx:45]: So let's RM where it's duplicative.

**Review by bruugey — APPROVED** [review]

**Pull request merged**